mmv1/third_party/terraform/services/gkehub2/resource_gke_hub_fleet_test.go.erb

```diff
@@ -74,6 +77,12 @@ resource "google_gke_hub_fleet" "default" {
   project = google_project.project.project_id
   display_name = "my staging fleet"
   default_cluster_config {
+	binary_authorization_config {
```

Comment:
It looks like skip_test is true on this test (so this field actually isn't being tested).
https://github.com/GoogleCloudPlatform/magic-modules/blob/d9e44319f1ee4d932731df6b783d8bfaee495840/mmv1/products/gkehub2/Fleet.yaml#L51


Could you add it to a test in the handwritten folder?
https://github.com/GoogleCloudPlatform/magic-modules/blob/d9e44319f1ee4d932731df6b783d8bfaee495840/mmv1/third_party/terraform/services/gkehub2/resource_gke_hub_fleet_test.go.erb#L1

Reply:
This default test isn't generated, but `mmv1/third_party/terraform/services/gkehub2/resource_gke_hub_fleet_test.go.erb` does get populated into the generated output (and subsequently tested)

* Path to output: $GOPATH/src/github.com/hashicorp/terraform-provider-google-beta/google-beta/services/gkehub2/resource_gke_hub_fleet_test.go

* Generated test: https://paste.googleplex.com/5902079424462848?raw